client/src/main/java/gui/controllers/PlayerSceneController.java

```diff
@@ -14,22 +16,25 @@
 import javafx.stage.Stage;
 import javafx.stage.Window;
 
+import static gui.utility.CommandType.SEND_PLAYER;
+
 public class PlayerSceneController {
 
+    private static final int PORT_NUMBER = 7777;
     @FXML
     private ComboBox<ServerInfo> serverNameComboBox;
     @FXML
     private TextField playerNameField;
     @FXML
     private Button startPlacement;
-
     private ClientAppRunner appInstance;
-    private ObservableList<ServerInfo> listOfServers;
-    private static final int PORT_NUMBER = 7777;
 
     public void initialize() {
         appInstance = ClientAppRunner.getInstance();
-        listOfServers = FXCollections.observableArrayList(ServerInfo.create("localhost", PORT_NUMBER));
+
+        ObservableList<ServerInfo> listOfServers = FXCollections
+                .observableArrayList(ServerInfo.create("localhost", PORT_NUMBER));
```

Comment:
Introduce constant with "localhost". In the future it will be simpler to change this statement.